Modules/Numerics/Optimizers/src/itkExhaustiveOptimizer.cxx

```diff
@@ -31,6 +31,8 @@ ExhaustiveOptimizer::ExhaustiveOptimizer()
   m_CurrentValue = 0;
   m_CurrentParameter = 0;
   m_CurrentIndex.Fill(0);
+  m_MaximumNumberOfIterations = 1;
+  m_MaximumMetricValue = itk::NumericTraits<MeasureType>::max();
```

Comment:
Please consider placing this in a constructor member initializer list:
 
     :
     m_MaximumNumberOfIterations(1),
     m_MaximumMetricValue(itk::NumericTraits<MeasureType>::max())

Reply:
Or (preferably) C++11 in-class default member initialization, in the header file:

Replace:
https://github.com/InsightSoftwareConsortium/ITK/blob/cde72b3e7d716542b11d2dbb402bbe5816cf731f/Modules/Numerics/Optimizers/include/itkExhaustiveOptimizer.h#L157-L159
With:

    SizeValueType m_MaximumNumberOfIterations{ 1 };

    MeasureType m_MaximumMetricValue{ itk::NumericTraits<MeasureType>::max() };

Reply:
Done in f1a52f1.

Reply:
@jhlegarreta Thank you, Jon! 👍